src/handlers/backport.rs

```diff
@@ -0,0 +1,227 @@
+use std::collections::HashMap;
+use std::sync::LazyLock;
+
+use crate::config::BackportTeamConfig;
+use crate::github::{IssuesAction, IssuesEvent, Label};
+use crate::handlers::Context;
+use regex::Regex;
+use tracing as log;
+
+// see https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue
+static CLOSES_ISSUE_REGEXP: LazyLock<Regex> =
+    LazyLock::new(|| Regex::new("(?i)(close[sd]*|fix([e]*[sd]*)?|resolve[sd]*) #(\\d+)").unwrap());
```

Comment:
I've seen contributor (probably my-self included using) `Fixes (after beta-backport) #123`). How do we want to take into account that extra marker?

Reply:
Yeah, I saw also Michael doing that. To my understanding it is to block GH autoclosing the issue, correct?

Your observation makes sense. But that syntax can be very aleatoric (`Fixes basically-anything-I-can-think-of #123`), I would probably think about it for a second before adding to the regexp :thinking: 

I'll make a note to discuss this in a second patch, would it be fine for you?

Reply:
I would just like to be sure avoiding introducing *false* triggers

Reply:
> at first sight I don't see a specific event for "this PR was linked to an issue".

I don't think there is one, I'm pretty sure we'll have to listen to the "Edited" event, which might not be that bad if we are clever and only analyze the changes (that should avoid unnecessary checking of labels when the fixes hasn't changed).

Reply:
I'm not sure how finding the cross-references with the Timeline api helps (since it's not a webhook), but sure, we can do that later.

Reply:
Ah Ok, then maybe I'm just overthinking it. I don't expect the opening comment to be edited *that* often. I'll just handle `IssuesEvent::{Opened|Edited}` :+1: 

When editing a comment the events are different (`IssueCommentEvent`).